src/v2/Apps/Conversation/Components/Conversation.tsx

```diff
@@ -85,45 +83,6 @@ const Conversation: React.FC<ConversationProps> = props => {
     })
   }
 
-  const messageGroups = groupMessages(
```

Comment:
To see where (and why) this code moved read 👉 [this](https://github.com/artsy/force/pull/6108/files#r472682665)